bots/machine/testvm.py

```diff
@@ -462,15 +462,15 @@ def execute(self, command=None, script=None, input=None, environment={},
                             proc.stdout.close()
                         else:
                             if self.verbose:
-                                sys.stdout.write(data.decode('utf-8', 'ignore'))
+                                os.write(sys.stdout.fileno(), data)
                             output += data.decode('utf-8', 'replace')
```

Comment:
Is this correct, or can it crash as well with "wrong" data?

Reply:
It cannot crash any more. Of course what can happen is that you just see garbage if there actually is garbage in the journal - it won't be filtered out any more. But I think that's still preferrable, and (1) in general the journal should be human readable text, and (2) if there is garbage then that's usually an actual bug to fix elsewhere. But it shouldn't break our tests.

In this case it's fancy quotes:
```
+ journalctl 2>&1 -o cat -p 5 SYSLOG_IDENTIFIER=cockpit-ws SYSLOG_IDENTIFIER=cockpit-bridge || true
error loading contents of os-release: Failed to open file “/usr/lib/os-release”: No such file or directory
```